code/header_units/solution/Makefile

```diff
@@ -0,0 +1,25 @@
+all: header_units
+
+clean:
+	rm -rf *.o header_units gcm.cache
+
+algorithm:
+	${CXX} -std=c++20 -fmodules-ts -x c++-system-header $@
+
+numeric:
+	${CXX} -std=c++20 -fmodules-ts -x c++-system-header $@
+
+random:
+	${CXX} -std=c++20 -fmodules-ts -x c++-system-header $@
+
+vector:
+	${CXX} -std=c++20 -fmodules-ts -x c++-system-header $@
```

Comment:
Do you want to unify this to one target?

Reply:
We could, but these can be built in parallel. Can I express this in one target?

Reply:
I do not see why saying 
```Makefile
algorithm numeric random vector:
	${CXX} -std=c++20 -fmodules-ts -x c++-system-header $@
```
would not be strictly identical to the current code, and in particular would not allow to run in parallel. To be checked though, but it looks like pure syntax.